package.json

```diff
@@ -30,6 +30,8 @@
     "build": "babel ./src --out-dir ./dist --copy-files",
     "build:clean": "rimraf ./build",
     "build:watch": "babel ./src --out-dir ./dist --watch",
+    "examples:build" : "babel ./examples --out-dir ./dist/examples --ignore node_modules",
```

Comment:
I think our examples not for running, it's just a sample pseudo code, wich helps users to get main ideas. Otherwise, to run all examples will need to have azure etc stuff in package.json